site/book/06-deploying-packages/02-applying-a-package.md

```diff
@@ -58,14 +70,15 @@ would see that 6 resources would be created:
 
 ```shell
 $ kpt live apply wordpress --dry-run
-service/wordpress created (dry-run)
-service/wordpress-mysql created (dry-run)
-deployment.apps/wordpress created (dry-run)
-deployment.apps/wordpress-mysql created (dry-run)
-persistentvolumeclaim/mysql-pv-claim created (dry-run)
-persistentvolumeclaim/wp-pv-claim created (dry-run)
-6 resource(s) applied. 6 created, 0 unchanged, 0 configured, 0 failed (dry-run)
-0 resource(s) pruned, 0 skipped, 0 failed (dry-run)
+Dry-run strategy: client
```

Comment:
I am thinking if there is a good way to indicate the steps are running in dry-run mode. I could imagine myself mid-way freaking out if I *actually* applied or it's dry-run and I will have to scroll back to look at the command args.

The previous layout with `(dry-run` suffix wasn't terrible. What's the reasoning behind this change ?

Reply:
There have been some discussions around this. The previous output format is just very verbose, as the output on every line is identical, and in fact only prints the value of the flag provided through the flag. This also aligns the different output formats, as only the events output format had the dry-run suffix.